### PR TITLE
Increase clickable areas on mobile

### PR DESCRIPTION
**Pull request by drazik**
This increases some clickable area on mobile. Mostly on the mobile date select. It also adds a margin to the back button (the same as the cozy bar menu button), so the page title don't jump anymore.

### CODE REVIEW

File: src/components/SelectDates/SelectDates.jsx [hunk 1]
```diff
@@ -43,13 +43,21 @@ const capitalizeFirstLetter = string => {
 }
 
 const Separator = () => (
-  <Chip.Separator className={styles.SelectDates__separator} />
+  <Chip.Separator className={cx(styles.SelectDates__separator)} />
```
Comment (ptbrowne) [inline src/components/SelectDates/SelectDates.jsx]: cx call here is not used :)

File: src/components/SelectDates/SelectDates.jsx [hunk 2]
```diff
@@ -278,7 +286,7 @@ class SelectDatesDumb extends React.PureComponent {
         </span>
 
         <span className={styles.SelectDates__buttons}>
-          {isMobile && <Separator />}
+          {isMobile && <Separator noRightSpace />}
```
Comment (ptbrowne) [inline src/components/SelectDates/SelectDates.jsx]: not used finally ?

File: src/components/SelectDates/SelectDates.styl [hunk 3]
```diff
@@ -78,27 +74,24 @@ select-date-height-small = 2.75rem
...
     .SelectDates__buttons
         display flex
         align-items center
+        margin-left 0.375rem
 
     .SelectDates__Button
         width auto
         background-color transparent
         color coolGrey
+        padding-left 14px
```
Comment (ptbrowne) [inline src/components/SelectDates/SelectDates.styl]: 0.875rem no ?

File: src/components/SelectDates/SelectDates.jsx (hunk 2 above)
Comment (drazik) [reply src/components/SelectDates/SelectDates.jsx]: :+1:

File: src/components/SelectDates/SelectDates.styl (hunk 3 above)
Comment (drazik) [reply src/components/SelectDates/SelectDates.styl]: :+1:

File: src/components/SelectDates/SelectDates.jsx (hunk 1 above)
Comment (drazik) [reply src/components/SelectDates/SelectDates.jsx]: :+1:

**Pull request merged**